resources/testharness.js

```diff
@@ -792,8 +792,10 @@ policies and contribution forms [3].
 
     function done() {
         if (tests.tests.length === 0) {
-            tests.status.status = tests.status.ERROR;
-            tests.status.message = "done() was called without first defining any tests";
+            if (tests.status.status === null) {
```

Comment:
This extra check is a workaround for https://github.com/web-platform-tests/wpt/pull/20036#issuecomment-550237192 but it might change, I'll use whatever change @jugglinmike lands to fix it.